documentation/docs/workflow.md

```diff
@@ -0,0 +1,142 @@
+Maps to the major version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Breaking change`
+
+##### Added
+Maps to the minor version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Added`
+
+##### Patch
+Maps to the patch version of [Semantic Versioning](#Semantic-Versioning).
+labels: `Patch`
+
+##### Deprecated
+Marks features that will be removed in the feature. No special version consideration apply here, since the feature did not change yet.
+labels: `Deprecated`
+
+##### Removed
+Marks features as being removed. Typically the features should have been deprecated in the previous version. This maps to the major version of [Semantic Versioning](#Semantic-Versioning). 
+labels: `Removed`
+
+##### Excluded Tags
+Marks changes that should not be part of the changelog. This should only be used for documentation and rare exceptions
+labels: `Documentation`, `No changelog`
+
+Please note that a PR can only be listed in one section of a changelog. So attaching the labels `Rust` `Added` `Patch` to a PR is invalid.
+
+##### Release Summary
+To attach a release summary to a version in the changelog, an issue with the label `release-summary` must be created. Create a GitHub milestone matching the version you want to describe and attach it to the issue. The issue can be closed immediately. The text of the issue will be included in the changelog as the release summary.
+
+### Issue Labels
+The following labels are used to categorize issues but do not have any effect on changelogs: `Request`, `Enhancement`, `Bug`, `Chore`, `Dependencies`, `Help wanted`, `Duplicate`, `Wontfix`.
+
+## Release
+
+With the release process we can deliver versions of our software to the community. We use sensible automation where it helps to remove tedium. However, some steps that require active decision making remain manual.
+
+The final list of changes from the [changelog](#Changelog) informs the version of the release. If at least one change mapping to a major version is included, the major version needs to be incremented. In that case the minor and patch version are set to `0`. If there are no change related to a major version, but changes related to a minor version are present the minor version needs to be incremented while the patch version is set to `0`. Otherwise only the patch version is incremented. Determining the version of the release is the responsibility of the person performing the release.
+The determined version of the release is used to create the [hotfix](#hotfix) or [release](#release) branch. For example a major release from the previous version `v2.3.1` will create the `release/v3.0.0` branch.
+Notice the `v` in front of the version. We [tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) all release in git in the form of `vMAJOR.MINOR.PATCH`. For bindings we prefix the tag with the binding name, so a tag for Wasm would look like `wasm-v1.2.3`. Bindings and the core Rust library are versioned and released independently.
```

Comment:
```suggestion
The final list of changes from the [changelog](#Changelog) informs the version of the release. If at least one change mapping to a major version is included, the major version needs to be incremented. In that case the minor and patch versions are set to `0`. If there are no change related to a major version, but changes related to a minor version are present, the minor version needs to be incremented while the patch version is set to `0`. Otherwise only the patch version is incremented. Determining the version of the release is the responsibility of the person performing the release.

The determined version of the release is used to create the [hotfix](#hotfix) or [release](#release) branch. For example, a major release from the previous version `v2.3.1` will create the `release/v3.0.0` branch.
Notice the `v` in front of the version. We [tag](https://git-scm.com/book/en/v2/Git-Basics-Tagging) all release in git in the form of `vMAJOR.MINOR.PATCH`. For bindings we prefix the tag with the binding name, so a tag for Wasm would look like `wasm-v1.2.3`. Bindings and the core Rust library are versioned and released independently.
```